credential-viewer-component.js

```diff
@@ -141,13 +141,20 @@ function Ctrl(
   };
 
   self.confirmDeleteCredential = function(err, result) {
+    var credentialServiceOptions = {};
+    if(!compareHost(self.credential.id)) {
+      credentialServiceOptions.url = self.altUpdateEndpoint + '?id=' +
+        encodeURIComponent(self.credential.id);
+    }
     if(!err && result === 'ok') {
       self.credential.deleted = true;
-      // wait to delete so modal can transition
-      brCredentialService.collection.del(self.credential.id, {delay: 400})
+      brCredentialService.collection
+        .del(self.credential.id, credentialServiceOptions)
```

Comment:
This removed the delay, did we need that? I don't remember it's purpose despite the comment.